packages/ui/package.json

```diff
@@ -21,12 +21,13 @@
     "build": "yarn build:sd && yarn build:css && yarn build:js",
     "build:css": "./scripts/generateCSS.js",
     "build:js": "tsup --minify",
-    "build:sd": "style-dictionary build --config ./sd.config.js",
+    "build:sd": "ts-node --transpile-only sd.config.ts",
```

Comment:
Running Style Dictionary in Typescript

![cool](https://media.giphy.com/media/62PP2yEIAZF6g/giphy.gif)